docs/resources/workspace.md

```diff
@@ -0,0 +1,145 @@
+          rating_condition = {
+            field    = "RISK"
+            operator = "EQUAL"
+            values   = ["CRITICAL"]
+          }
+        },
+        {
+          operator = "AND"
+          string_condition = {
+            field    = "PLATFORM"
+            operator = "EQUAL"
+            values   = ["redhat", "debian"]
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `asset_selections` (Attributes List) A list of workspace selections. (see [below for nested schema](#nestedatt--asset_selections))
+- `name` (String) Name of the workspace.
+
+### Optional
+
+- `description` (String) Description of the workspace.
+- `space_id` (String) Mondoo space identifier. If there is no ID, the provider space is used.
+
+### Read-Only
+
+- `mrn` (String) The Mondoo resource name (MRN) of the workspace.
+
+<a id="nestedatt--asset_selections"></a>
+### Nested Schema for `asset_selections`
+
+Required:
+
+- `conditions` (Attributes List) A list of conditions for the selection. (see [below for nested schema](#nestedatt--asset_selections--conditions))
+
+<a id="nestedatt--asset_selections--conditions"></a>
+### Nested Schema for `asset_selections.conditions`
+
+Required:
+
+- `operator` (String) Operator determining how the condition is joined with the other conditions in the list. Valid values: `AND`, `AND_NOT`
+
+Optional:
+
+- `int_condition` (Attributes) A condition with values of type int. (see [below for nested schema](#nestedatt--asset_selections--conditions--int_condition))
+- `key_value_condition` (Attributes) A condition with values of type key:value. (see [below for nested schema](#nestedatt--asset_selections--conditions--key_value_condition))
+- `rating_condition` (Attributes) A condition with values of type int. (see [below for nested schema](#nestedatt--asset_selections--conditions--rating_condition))
+- `string_condition` (Attributes) A condition with values of type string. (see [below for nested schema](#nestedatt--asset_selections--conditions--string_condition))
+
+<a id="nestedatt--asset_selections--conditions--int_condition"></a>
+### Nested Schema for `asset_selections.conditions.int_condition`
+
+Required:
+
+- `field` (String) Numeric field to match. Valid values: ["RISK_SCORE"]
+- `operator` (String) Numeric operator. Valid values: ["EQUAL" "NOT_EQUAL" "GT" "LT"]
+- `values` (List of Number) Int values to match. Values are ORed together.
+
+
+<a id="nestedatt--asset_selections--conditions--key_value_condition"></a>
+### Nested Schema for `asset_selections.conditions.key_value_condition`
+
+Required:
+
+- `field` (String) key:value field to match. Valid values: ["LABELS" "ANNOTATIONS"]
+- `operator` (String) Rating operator. Valid values: ["CONTAINS"]
+- `values` (Attributes List) key:value list to match. Values are ORed together. (see [below for nested schema](#nestedatt--asset_selections--conditions--key_value_condition--values))
```

Comment:
```suggestion
- `values` (Attributes List) key:value list to match. Values are ORed together. (See [below for nested schema](#nestedatt--asset_selections--conditions--key_value_condition--values))
```

Reply:
All of these changes `see` > `See` are part of the code generator, I will try to see if I can change it.

Reply:
Oh rats! Well it would be great if you can... but of course it's not the end of the world. 😃